src/resources/resources.css

```diff
@@ -81,9 +82,77 @@
 	display: block;
 }
 
+/* TODO: Add pagination css */
 .pagination-container {
 	margin: auto;
+}
+
+/* List Buttons To Render Items from Database */
+.resource-list-item {
 	display: flex;
+	flex-direction: column;
+	padding-left: 0rem;
 }
 
-/* TODO: Add pagination css */
+.resource-list-title {
+	position: relative;
+	text-align: left;
+	display: block;
+	padding: 2rem 0.75rem 2rem 0.75rem;
+	color: white;
+	background-color: transparent;
+	border: 0;
+	border-bottom: 2px solid #ccc;
```

Comment:
No need to add border for both `title` and `description`. It's redundant. You just need to add border in `resources-list-item` which result same effect.

Reply:
Done! Needed to add an extra ruleset for ".resource-list-item button" to override default button styling.